client/js/socket-events/init.js

```diff
@@ -64,8 +64,7 @@ function openCorrectChannel(clientActive, serverActive) {
 
 	// Open window provided in location.hash
 	if (target.length === 0 && window.location.hash) {
-		target = $("#footer, #sidebar, #help")
-			.find(`[data-target="${escape(window.location.hash)}"]`);
+		target = $("body").find(`[data-target="${escape(window.location.hash)}"]`);
```

Comment:
Either use `document.body` or you can do `$(`[data-target="...` directly.

Reply:
You're right, will do the latter. I'll also add a `.first()` too, just in case.

Reply:
Done.